ios/RNSScreenStackHeaderConfig.h

```diff
@@ -1,13 +1,35 @@
-#import <React/RCTConvert.h>
+#ifdef RN_FABRIC_ENABLED
+#import <React/RCTViewComponentView.h>
+#import <UIKit/UIKit.h>
```

Comment:
Can we remove `UIKit` import?

Reply:
[1e30773](https://github.com/software-mansion/react-native-screens/pull/1417/commits/1e307737c0acd60cd2ba42f778d08044dae8c7e0)